rbac/management/audit_log/view.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""View for Audit Logs."""
+from management.models import AuditLog
+from management.serializers import AuditLogSerializer
+from rest_framework import mixins, viewsets
+from rest_framework.permissions import AllowAny
+
+
+class AuditLogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    """Audit Logs View.
+
+    A viewset that provides default `list()` actions.
+
+    """
+
+    queryset = AuditLog.objects.all()
```

Comment:
how this work ? and how this will be used ? Should not be incorporated `tenant_id`condition somehow ?
(Maybe I am just missing follow up code)

Reply:
I'm not entirely sure if this answers the question, but here is an example of what the get request looks like from when a role is created: 
![Screenshot from 2024-02-12 16-11-44](https://github.com/RedHatInsights/insights-rbac/assets/29558356/98093f0c-f4f1-4b52-858d-19e3c469e850)

Reply:
This is what it looks like in the database: 
![Screenshot from 2024-02-12 16-12-30](https://github.com/RedHatInsights/insights-rbac/assets/29558356/d904cb69-4aa8-42a1-b48d-4214b4cb5601)

Reply:
Do you mean to list the tenant ID for the get request? 

Thank you so much!

Reply:
I was thinking when queryset is called and  if it would result in retrieving all records from auditlogs on such a request. If that's the case, I would certainly consider adding a condition for tenant_id=XX in some way

Reply:
Oh! When the queryset is called, it would result in retrieving all the records from audit logs! 

Would that condition be related to filtering the query set to just list the logs that are related to a specific tenant id or something different? 

Thank you so much!